.circleci/config.yml

```diff
@@ -0,0 +1,57 @@
+# Inspired by: http://mikebian.co/running-tests-against-multiple-ruby-versions-using-circleci/
+
+version: 2.1
+
+orbs:
+  ruby: circleci/ruby@1.1
+
+jobs:
+  test:
+    parallelism: 1
+    parameters:
+      ruby-image:
+        type: string
+    docker:
+      - image: << parameters.ruby-image >>
+      - image: rabbitmq
+
+    steps:
+      - checkout
+      - run:
+          name: install dockerize
+          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && sudo tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          environment:
+            DOCKERIZE_VERSION: v0.3.0
+      # - run:
```

Comment:
After setting up [circle for the project](https://app.circleci.com/pipelines/github/mxenabled/action_subscriber/1/workflows/e74903ca-7383-4e24-9ce3-1805ee519f91/jobs/2) with this config file, it seems we are getting a failure trying to talk to rabbit's http api on `15672`.